layers/+chat/circe/README.org

```diff
@@ -0,0 +1,84 @@
+#+TITLE: Circe Layer
+
+* Table of Contents                                         :TOC_4_gh:noexport:
+- [[#description][Description]]
+- [[#features][Features]]
+- [[#install][Install]]
+  - [[#layer][Layer]]
+    - [[#helm-auto-completion-for-usernames][Helm auto-completion for usernames]]
+  - [[#default-servers][Default servers]]
+    - [[#security-note][Security Note]]
+- [[#key-bindings][Key bindings]]
+- [[#spacemacs-layout-support][Spacemacs Layout Support]]
+
+* Description
+Layer for [[http://www.emacswiki.org/emacs/Circe][Circe IRC chat]]. 
```

Comment:
Needs a "the", and should say "client": "the Circe IRC client".

The wiki says that https://github.com/jorgenschaefer/circe is the homepage for Circe.  Is there a reason to link to the wiki instead of the GitHub page?